metadata-ingestion/src/datahub/cli/config_utils.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import os
+import sys
+from typing import Optional, Union
+
+import click
+import yaml
+from pydantic import BaseModel, ValidationError
+
+from datahub.cli.env_utils import get_boolean_env_variable
+
+__help__ = (
+    "For helper methods to contain manipulation of the config file in local system."
```

Comment:
module level docstrings should go at the top and be in `"""`, don't need to use the `__help__` thing

Reply:
Going to do this in follow up PR.